docs/topics/running/services/rate-limit-service.md

```diff
@@ -53,11 +60,51 @@ spec:
 
 ## External Rate Limit Service
 
-In order for Ambassador Edge Stack to rate limit, you need to implement a gRPC `RateLimitService`, as defined in [Envoy's `rls.proto`][rls.proto] interface. If you do not have the time or resources to implement your own rate limit service, Ambassador Edge Stack integrates a high performance, rate limiting service.
-
-[rls.proto]: https://github.com/datawire/ambassador/tree/master/api/envoy/service/ratelimit/v2/rls.proto
-
-Ambassador Edge Stack generates a gRPC request to the external rate limit service and provides a list of labels on which the rate limit service can base its decision to accept or reject the request:
+In order for the Ambassador API Gateway to rate limit, you need to implement a
+gRPC `RateLimitService`, as defined in [Envoy's `v1/rls.proto`][`v1/rls.proto`]
+interface.  If you do not have the time or resources to implement your own rate
+limit service, the Ambassador Edge Stack integrates a high-performance rate
+limiting service.
+
+> Note: *In a future version of Ambassador*, the Ambassador API Gateway will
+> change the version of the gRPC service name used to communicate
+> `RateLimitService`s from the one defined in [`v1/rls.proto`][]
+> (`pb.lyft.ratelimit.RateLimitService`) to the one defined in
+> [`v2/rls.proto`][] (`envoy.service.ratelimit.v2.RateLimitService`):
+>
+> - In some future version of Ambassador, there will be a setting to control
+>   which name is used; with the default being the current name; it will be
+>   opt-in to the new name.
+>
+> - In some future version of Ambassador after that, *no sooner than Ambassador
+>   1.6.0*, the default value of that setting swill change; making it opt-out
+>   from the new name.
```

Comment:
`s/swill/will/` -- and I'd probably make this "no sooner than 1.7.0" at this point.